Assets/Scripts/Game/TalkManager.cs

```diff
@@ -2557,11 +2557,15 @@ private void AssembleTopicListPerson()
                             }
                         }
 
-                        if (!dialogPartnerIsSamePersonAsPersonResource && // only make it available for talk if dialog partner is not the same person as the person resource talked about
-                            questResourceInfo.Value.availableForDialog && // only make it available for talk if it is not "hidden" by dialog link command
-                            questResourceInfo.Value.hasEntryInWhereIs &&
-                            IsPlayerInSameLocationWorldCell) // only make it available for talk if person resource is in same world map location as player
+                        // only make it available for talk if...
+                        if (!dialogPartnerIsSamePersonAsPersonResource &&   // dialog partner is not the same person as the person resource talked about
+                            questResourceInfo.Value.availableForDialog &&   // it is not "hidden" by dialog link command
+                            questResourceInfo.Value.hasEntryInWhereIs &&    // it is meant to have an entry in the where is section (currently this is always true, TODO: check if we can get rid of this)
+                            !person.IsHidden &&                             // if person resource is not hidden
```

Comment:
This is the bit that worries me. From "hide npc" in Tipton's docs:

> Hidden NPCs are invisible to the player character, but any Qrc messages associated with that NPC will work properly; the PC is just unable to see the NPC anywhere in the game world.

Overall, we might actually be better off allowing the early discovery of thief's location that risk kicking the can down the road and breaking something else by changing the intent of "hide npc" to include talk system.

Reply:
tipton is wrong about dialog in 50% of what he is saying - I can elaborate on this on ten+ pages (and I will)
but no not 100% confident but we can change this later if it makes problems
another fix for A0C00Y12 would be not to place npc at all like in the mages guild quest - let me know

Reply:
e.g. he says that rumors at person resource line creates an entry for "any news" (this is wrong - I tested this extensively)

Reply:
I will test the behavior of hide npc in classic later tonight it tomorrow.
Tbh it looks like a bug to me if it does not hide dialog, we will see. I will also take a look how often this action is used and see if it can break those quests